recipe/meta.yaml

```diff
@@ -18,10 +18,12 @@ requirements:
   host:
     - pip
     - python >=3.7
+    - pytorch >=1.12.0
```

Comment:
```suggestion
    - pytorch >=1.12.0
    - pybind11
    - aws-sdk-cpp
```

Reply:
There is an issue with `pybind11` and `aws-sdk-cpp` being registered as .gitmodules in the original repo. The build fails when trying to pull them. Both dependencies are part of conda-forge, so maybe we can just add them in the meta.yaml ?

Would also require to set `USE_SYSTEM_LIBS=True` in the build script to toggle off the submodule pull.

Reply:
Yes, that makes sense. Sorry, haven‘t had time yet to investigate. 
Do you by chance know if that is the only thing that is being compiled otherwise or is something in torchdata itself also being compiled?

Reply:
I don't really know unfortunately. But I could successfully run a local build with those changes.

Reply:
That seems to have done the trick. I'll try it locally to see if it is noarch or not and then I'll merge. Thanks!